global.json

```diff
@@ -19,14 +19,14 @@
     "Git": "2.22.0",
     "jdk": "11.0.3",
     "vs": {
-      "version": "16.5",
+      "version": "16.7",
```

Comment:
This is a small increment past what is available on the main CI agents. @wtgodbe or @pranavkm (I think you're on call tomorrow) if I get this in tonight or early-ish tomorrow, could you please test a draft PR on the BuildPool.Server.Amd64.VS2019.Pre.Open queue❔ I'd rather things didn't fall over once 16.7 reaches the main 2019 queue without us preparing.